image/lxd.go

```diff
@@ -37,12 +37,17 @@ func NewLXDImage(sourceDir, targetDir, cacheDir string,
 }
 
 // Build creates a LXD image.
-func (l *LXDImage) Build(unified bool, compression string) error {
+func (l *LXDImage) Build(unified bool, compression string, vm bool) error {
 	err := l.createMetadata()
 	if err != nil {
 		return nil
 	}
 
+	if vm {
+		// If we're building a VM, we need the separate metadata tarball.
+		unified = false
```

Comment:
Technically unified is possible with a VM, it would be a single tarball containing the metadata files and templates and a `rootfs.img` file (qcow2) instead of the `rootfs` directory.